code/game/objects/items/shields.dm

```diff
@@ -61,6 +63,16 @@
 		if(0 to 25)
 			. += span_warning("It's falling apart!")
 
+/obj/item/shield/attackby(obj/item/attackby_item, mob/user, params)
+	. = ..()
+	if(istype(attackby_item, /obj/item/melee/baton) && is_bashable)
+		if(!COOLDOWN_FINISHED(src, baton_bash))
+			return
+		user.visible_message(span_warning("[user] bashes [src] with [attackby_item]!"))
+		playsound(user.loc, 'sound/effects/shieldbash.ogg', 50, TRUE)
+		COOLDOWN_START(src, baton_bash, BATON_BASH_COOLDOWN)
+		return
```

Comment:
```suggestion
	if(!istype(attackby_item, /obj/item/melee/baton) || !is_bashable)
		return
		
	if(!COOLDOWN_FINISHED(src, baton_bash))
		return
	user.visible_message(span_warning("[user] bashes [src] with [attackby_item]!"))
	playsound(user.loc, 'sound/effects/shieldbash.ogg', 50, TRUE)
	COOLDOWN_START(src, baton_bash, BATON_BASH_COOLDOWN)
```

early return to lessen bad-looking indentation, also that return on the end is totally superfluous

Reply:
wait what, baton bash? how the hell did that commit made it's way here. I really need to fix my fork